gammapy/spectrum/tests/test_models.py

```diff
@@ -1,6 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 import pytest
 import numpy as np
+from naima import models, radiative
```

Comment:
shouldn't there be a `@require_dependency("naima")` to avoid some build fails?

Reply:
also you should call naima specific code in a way that is clearly different from spectrum.models.
e.g. `import naima`  and later `naima.models` and `naima.radiative`
or `from naima import models as naima_models`

Reply:
Thank you! I apply all the changes you suggest

Reply:
The `@require_dependency("naima")` decorator should be just used for the `test_models()` function?